MANIFEST.in

```diff
@@ -1,3 +1,5 @@
 include README.md
 include LICENSE
-include pydrad/configure/templates/*
\ No newline at end of file
+include pyproject.toml
```

Comment:
I think this file shold be empty unless you need to prune.

Between setuptools_scm and I think the data finder in setup.cfg, that should cover all of this. See https://setuptools.pypa.io/en/latest/userguide/datafiles.html

Reply:
You mean the whole manifest file should be empty?

Reply:
You should be able to delete the file.